src/services/localesServices/locales/en.json

```diff
@@ -123,5 +123,9 @@
 	"rejected": "rejected",
 	"approved": "approved",
 	"that you requested has been": "that you requested has been",
-	"To view this %s request you made click below": "To view this %s request you made click below"
+	"To view this %s request you made click below": "To view this %s request you made click below",
+	"Request with id 51e74db7-5510-4f50-9f15-e23710331ld5 has been edited": "Request with id 51e74db7-5510-4f50-9f15-e23710331ld5 has been edited",
+	"To view this edited request click below": "To view this edited request click below",
+	"Request with id 5cb6bd43-f4ab-4fd9-86c6-37075e4f7957 has been edited": "Request with id 5cb6bd43-f4ab-4fd9-86c6-37075e4f7957 has been edited",
```

Comment:
could you remove these unused messages?

Reply:
they were being added when I ran tests but I removed them